### PR TITLE
Bump cert-exporter container image to v4.5

### PR DESCRIPTION
**Pull request by jenting**
xrefs https://github.com/SUSE/avant-garde/issues/1912

### CODE REVIEW

**Comment by jenting** [comment]
@cclhsu @innobead @c3y1huang PTAL :bow:

File: stable/cert-exporter/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The following table lists the configurable parameters of the cert-exporter chart
 
 | Parameter                            | Description                                                                       | Default                                    |
 | ------------------------------------ | --------------------------------------------------------------------------------- | ------------------------------------------ |
-| `image.repository`                   | The image repository to pull from                                                 | `registry.suse.com/caasp/v4/cert-exporter` |
+| `image.repository`                   | The image repository to pull from                                                 | `registry.suse.com/caasp/v4.5/cert-exporter` |
```
Comment (c3y1huang) [inline stable/cert-exporter/README.md:33]: nit: could you help to align this?
Screenshot-2020-08-27-15:40:45

**Review by c3y1huang — APPROVED** [review]

**Comment by c3y1huang** [comment]
Maybe could also update step 1 and 2 so they start with a Capital letter.
```
1. Monitor cert-manager issues certificates in cert-managert-test namespace.
2. Monitor certificates in all namespaces filtered by label selector.
```

**Comment by jenting** [comment]
> Maybe could also update step 1 and 2 so they start with a Capital letter.
> 
> ```
> 1. Monitor cert-manager issues certificates in cert-managert-test namespace.
> 2. Monitor certificates in all namespaces filtered by label selector.
> ```

sure~

**Comment by c3y1huang** [comment]
Need to bump up chart version too.

**Review by innobead — APPROVED** [review]

**Review by cclhsu — APPROVED** [review]

**Comment by cclhsu** [comment]
@maximenoel8 Can you help test this if ok to merge? I merged it accidentally.

**Pull request merged**